applicationconfig/client/src/main/java/com/azure/applicationconfig/AzConfigClient.java

```diff
@@ -31,6 +31,7 @@
 import com.microsoft.rest.v3.annotations.QueryParam;
 import com.microsoft.rest.v3.annotations.UnexpectedResponseExceptionType;
 import com.microsoft.rest.v3.http.HttpPipeline;
+import com.microsoft.rest.v3.http.policy.HttpLogDetailLevel;
```

Comment:
Need to reconcile this with @anuchandy - I'm not sure if the plan is for this class to go away or not.

Reply:
@JonathanGiles created an issue to discuss this https://github.com/Azure/autorest-clientruntime-for-java/issues/577 based on your second set of review feedback.

While this is very useful for customers and also for us to debug customer issues, if we can improve the experience then we should.